CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## v11.1.0.0
+
+* Upgrade to Cumulus [v11.1.0](https://github.com/nasa/Cumulus/releases/tag/v11.1.0)
+  * see [Cumulus v11.0.0](https://github.com/nasa/Cumulus/releases/tag/v11.0.0) release notes for required migration steps for workflows and collection configurations
```

Comment:
I want to point out something about how the v9.9 -> v11 migration steps are the same steps as v9.9 -> v10.1.2, but the "after cumulus deployment section" of v11.0.0 (which discusses the `type` column and running a sizable migration) will still be relevant for someone who has upgraded from v9.9 to v10.1.2 and then wants to go to v11 (as we have done at NSIDC).

Maybe something like this?

```suggestion
  * see [Cumulus v11.0.0](https://github.com/nasa/Cumulus/releases/tag/v11.0.0) release notes for required migration steps for workflows and collection configurations, as well as lambda executions. If upgrading from CIRRUS v9.9.0.0 or an earlier version, see the v10.1.2.0 notes as well.
```